packages/@aws-cdk/aws-lambda/test/event-source-mapping.test.ts

```diff
@@ -223,7 +223,7 @@ describe('event source mapping', () => {
       target: fn,
       eventSourceArn: eventSourceArn,
       kafkaTopic: topicNameParam.valueAsString,
-      filters: [
+      filterCriteria: [
```

Comment:
Lets keep at least one old test for `filters`, because again, that behavior has to be the same as before.